src/pages/hero/Hero.jsx

```diff
@@ -1,94 +1,80 @@
-                alt="hero"
-                className="h-full w-full"
-              />
-            </div>
-            <div className="h-36 w-64 bg-gray-800 rounded-lg overflow-hidden">
-              <img
-                src="https://picsum.photos/200/200"
-                alt="hero"
-                className="h-full w-full"
-              />
-            </div>
-            <div className="h-36 w-64 bg-gray-800 rounded-lg overflow-hidden">
-              <img
-                src="https://picsum.photos/200/200"
-                alt="hero"
-                className="h-full w-full"
-              />
-            </div>
-            <div className="h-36 w-64 bg-gray-800 rounded-lg overflow-hidden">
-              <img
-                src="https://picsum.photos/200/200"
-                alt="hero"
-                className="h-full w-full"
-              />
-            </div>
-            <div className="h-36 w-64 bg-gray-800 rounded-lg overflow-hidden">
-              <img
-                src="https://picsum.photos/200/200"
-                alt="hero"
-                className="h-full w-full"
-              />
-            </div>
-            <div className="h-36 w-64 bg-gray-800 rounded-lg overflow-hidden">
-              <img
-                src="https://picsum.photos/200/200"
-                alt="hero"
-                className="h-full w-full"
-              />
-            </div>
+          <div className="flex justify-center py-8 flex-wrap gap-8">
+            {categories &&
+              categories.map((category) => (
+                <div
+                  key={category.id}
+                  onClick={() =>
+                    Navigate(`/explore/?category/${category.categoryName}`)
+                  }
+                  className="relative w-72 h-96 cursor-pointer will-change-auto rounded-xl overflow-hidden hover:border-4 border-rose-400 shadow-lg transition-border duration-100 ease-in-out "
+                >
+                  <CategoryCard category={category} />
+                </div>
+              ))}
           </div>
         </section>
+        <h1 className="text-2xl kodchasan uppercase text-center py-12  text-gray-400 font-bold">
+          Must watch videos
+        </h1>
+        <section>
+          {categories &&
+            categories.map((category) => {
+              const getVideoByCategory = categoryFilter(
+                videos,
+                category.categoryName
+              );
+              return (
+                <div key={category._id}>
+                  <h2 className="text-xl kodchasan uppercase text-left px-8 py-4  text-orange-300 font-bold">
+                    {category.categoryName}
+                  </h2>
+                  <section>
+                    <div className="flex justify-center py-8 flex-wrap gap-8">
+                      {getVideoByCategory &&
+                        getVideoByCategory.slice(0, 8).map((video) => (
```

Comment:
Suggestion: use variable for this

Reply:
can't do this in a variable, since `getVideoByCategory` is a function and inside that function, I am passing `video` as an argument that is coming up after `videos` is mapped. To access video value inside the function I cant place it outside the mapped scope (videos).